### ISSUE TITLE
Workflow for translations

### ISSUE THREAD

**Issue by gannebamm** [GeoNodeUserGroup-DE/geonode-blueprint-docker#2]
What is the workflow for updating translations? We have to work with two separate repositories, right?

geonode-mapstore-client and the Django-based translations?

**Comment by ridoo** [comment]
Yes, there are still translation files for GeoNode and geonode-MapStore-client. However, I do not think the only way would be to make changes in the two repos. I am pretty sure, there is a way to add or override translations within the blueprint. I will have to check and add some examples how this can be done.

**Comment by gannebamm** [comment]
see working example here: https://github.com/Thuenen-GeoNode-Development/thuenen_atlas/pull/64

---
done by:

1.    working on the templates
 2.   execing into the django docker container, switching to the thuenen_app django app directory and using:
    `/usr/src/geonode/thuenen_app# DJANGO_SETTINGS_MODULE=geonode.settings python ../manage.py makemessages -l de --no-location --no-obsolete`
  3.  do create the .po file and
    `/usr/src/geonode/thuenen_app# DJANGO_SETTINGS_MODULE=geonode.settings python ../manage.py compilemessages -l de`
    to compile the .mo file.
